src/content/app/genome-browser/hooks/useBrowserRouting.test.tsx

```diff
@@ -178,18 +181,18 @@ describe('useBrowserRouting', () => {
 
   describe('navigation to /genome-browser/:genome_id', () => {
     it('sets the data from the url in redux', () => {
-      // jest.spyOn(browserActions, 'setDataFromUrlAndSave');
+      // jest.spyOn(browserEntityActions, 'setDataFromUrlAndSave');
       renderComponent({ path: '/genome-browser/human' });
 
-      expect(browserActions.setDataFromUrlAndSave).toHaveBeenCalledWith({
+      expect(browserEntityActions.setDataFromUrlAndSave).toHaveBeenCalledWith({
         activeGenomeId: 'human',
         activeEnsObjectId: null,
         chrLocation: null
       });
     });
 
     it('fetches the genome using the genome id from the url', async () => {
-      // jest.spyOn(browserActions, 'setDataFromUrlAndSave');
+      // jest.spyOn(browserEntityActions, 'setDataFromUrlAndSave');
```

Comment:
❓

Reply:
Hmm... I just renamed all `browserActions` -> `browserEntityActions`. 
I've removed it